models/renalStructures_UNEST_segmentation/docs/README.md

```diff
@@ -0,0 +1,81 @@
+# Description
```

Comment:
perhaps add a section about 'if you use the models please cite...'. if some code is from a different open-source repo, I think the original licenses should be included properly, for example https://github.com/Project-MONAI/MONAI/blob/a9095d09d919e94c2c07aa74fc46c1e4f9530ecd/monai/apps/detection/utils/box_coder.py#L12-L45

Reply:
Yes, I will add the section on thhe external code base.

Reply:
Added section of license description and additional citations.